src/swsssdk/interface.py

```diff
@@ -298,6 +298,19 @@ def get(self, db_name, _hash, key):
             # redis only supports strings. if any item is set to string 'None', cast it back to the appropriate type.
             return None if val == b'None' else val
 
+
+    @blockable
+    def hexists(self, db_name, _hash, key):
```

Comment:
>hexists [](start = 8, length = 7)

swsssdk is going to be deprecate in future. The same functionality is supported by [https://github.com/Azure/sonic-swss-common/blob/master/common/dbinterface.cpp](https://github.com/Azure/sonic-swss-common/blob/master/common/dbinterface.cpp)
and [https://github.com/Azure/sonic-swss-common/blob/master/common/sonicv2connector.cpp](https://github.com/Azure/sonic-swss-common/blob/master/common/sonicv2connector.cpp).
They are implemented in C++, and with a SWIG wrapper, it could be used in python project.

New python code should use sonic-swss-common instead of swsssdk.
Could you move your contribution there?